.github/workflows/publish.yml

```diff
@@ -0,0 +1,28 @@
+name: Publish Plugin on Tag
+
+on:
+  push:
+    tags:
+      - '[0-9]+.[0-9]+.[0-9]+'
+
+jobs:
+  build-plugin:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
```

Comment:
is checkout required here?

Reply:
No, it's not.  But should help traceability and richer logs..